proxy/http2/Http2Stream.cc

```diff
@@ -327,7 +327,12 @@ Http2Stream::do_io_close(int /* flags */)
     if (parent && this->is_client_state_writeable()) {
       // Make sure any trailing end of stream frames are sent
       // Wee will be removed at send_data_frames or closing connection phase
-      static_cast<Http2ClientSession *>(parent)->connection_state.send_data_frames(this);
+      Http2SendDataFrameResult result = static_cast<Http2ClientSession *>(parent)->connection_state.send_data_frames(this);
+      if (result == Http2SendDataFrameResult::NO_WINDOW) {
+        Http2StreamDebug("cancel closing");
+        closed = false;
```

Comment:
`super::release(nullptr)` is already called at the beginning of `do_io_close`. Wouldn't it be a problem?

Reply:
It looks like do nothing now. But it could be problem in the future. I'll move this call.
https://github.com/apache/trafficserver/blob/4d4283ea26f709fc9a7efea3283f1acc401bab0a/proxy/ProxyClientTransaction.cc#L71
https://github.com/apache/trafficserver/blob/4d4283ea26f709fc9a7efea3283f1acc401bab0a/proxy/http2/Http2ClientSession.h#L232-L235